### PR TITLE
Add clarifying note about _resolution_ in the most common case.

### PR DESCRIPTION
**Pull request by rohan-wire**
In the most basic case, the resolution of a node is the node, so let's say that.

### CODE REVIEW

**Comment by Bren2010** [comment]
Fwiw this is stated immediately below your note:
```
* The resolution of a non-blank node comprises the node itself,
followed by its list of unmerged leaves, if any
```

**Comment by rohan-wire** [comment]
Bren2010 said:
>Fwiw this is stated immediately below your note:

Yes. And it took me three readings to understand that the most complicated scenario was being described first and the most basic scenario is not explicitly stated. Most readers will not understand in what cases there will be unmerged leaves by this point.

File: draft-ietf-mls-protocol.md [hunk 1]
```diff
@@ -810,6 +810,7 @@ present are laid out in {{views}}.
 A node in the tree may also be _blank_, indicating that no value is present at
 that node.  The _resolution_ of a node is an ordered list of non-blank nodes
 that collectively cover all non-blank descendants of the node.  The resolution
+of a non-blank node with no unmerged leaves is just the node itself. The resolution
```
Comment (bifurcation) [inline draft-ietf-mls-protocol.md:813]:
```suggestion
of a non-blank node with no unmerged leaves is just the node itself. More generally, the resolution
```

**Review by bifurcation — APPROVED** [review]
Looks good with one minor edit.

**Pull request merged**